boards/BMS/Src/App/states/App_AllStates.c

```diff
@@ -140,19 +145,20 @@ bool App_AllStatesRunOnTick100Hz(struct StateMachine *const state_machine)
         App_Airs_IsAirNegativeClosed(airs) ? CONTACTOR_STATE_CLOSED : CONTACTOR_STATE_OPEN);
     App_CanTx_BMS_Contactors_AirPositive_Set(
         App_Airs_IsAirPositiveClosed(airs) ? CONTACTOR_STATE_CLOSED : CONTACTOR_STATE_OPEN);
-
     App_SetPeriodicCanSignals_Imd(imd);
+
     App_AdvertisePackPower(accumulator, ts);
 
     App_CanTx_BMS_OkStatuses_BmsOk_Set(App_OkStatus_IsEnabled(bms_ok));
     App_CanTx_BMS_OkStatuses_ImdOk_Set(App_OkStatus_IsEnabled(imd_ok));
     App_CanTx_BMS_OkStatuses_BspdOk_Set(App_OkStatus_IsEnabled(bspd_ok));
 
-    const bool dcm_fault              = App_CanAlerts_BoardHasFault(DCM_ALERT_BOARD);
-    const bool fsm_fault              = App_CanAlerts_BoardHasFault(FSM_ALERT_BOARD);
-    const bool pdm_fault              = App_CanAlerts_BoardHasFault(PDM_ALERT_BOARD);
-    const bool dim_fault              = App_CanAlerts_BoardHasFault(DIM_ALERT_BOARD);
-    const bool fault_from_other_board = dcm_fault || fsm_fault || pdm_fault || dim_fault;
+    const bool dcm_fault = App_CanAlerts_BoardHasFault(DCM_ALERT_BOARD);
```

Comment:
Are there any faults we can have while charging that would be dangerous if not detected from other boards?

Reply:
Charging is a process that is handled solely by the BMS, no other boards are involved, so there shouldn't be any need to check for faults from other boards while charging specifically. (additionally charging is often done outside the car where the other boards aren't connected) Hence why I added a conditional operator on `fault_from_other_board` on line 161

@gtaharaedmonds did I get that right?

Reply:
That does make sense, if that's the case then this file should be good. I'll wait for some confirmation though

Reply:
I confirmed with @Eddy13013 that other boards will always be disconnected when charging